app/pages/lab/project.cjsx

```diff
@@ -252,7 +252,7 @@ module.exports = React.createClass
           <p className="form-help">Loading project</p>
         </div>
       } then={([project, owners]) =>
-        if @props.user in owners
+        if @props.user in owners or @props.user.admin
```

Comment:
I see. Thanks for the explanation. And presumably `apiClient.params.admin` also tells you they are an admin AND have it turned on (since they couldn't have turned it on otherwise)